commit 8391a075c8d83f3fcaf0dda6fe60bfb9200a1b4f (refs/changes/15/1415/1)
Author: andrew <andrew@whartonlabs.com>
Date:   2017-02-03T11:42:40+11:00 (2 years, 8 months ago)
    
    Fixed line length > 80 chars which 'make check' was complaining about

diff --git a/lib/tritonapi.js b/lib/tritonapi.js
index 848f5f7..b8b0bd2 100644
--- a/lib/tritonapi.js
+++ b/lib/tritonapi.js
@@ -2287,7 +2287,8 @@ TritonApi.prototype.deletePolicy = function deletePolicy(opts, cb) {
  *
  * @param {Object} opts
  *      - {String} id: Required. The instance name, short id, or id (a UUID).
- *      - {String} package: Required. The new package name, shortId or id (a UUID).
+ *      - {String} package: Required. The new package name, shortId
+ *        or id (a UUID).
  *      - {Boolean} wait: Wait (via polling) until the rename is complete.
  *        Warning: A concurrent resize of the same instance can result in this
  *        polling being unable to notice the change. Use `waitTimeout` to
